### PR TITLE
Document new repository configurations

### PR DESCRIPTION
**Pull request by BerrevoetsRobbe**
**To what scenario is your documentation request related? Please describe.**
The configuration of a repository has changed from conf files to yaml files. I wish to have clear documentation on the different parameters possible

**Id**
8a47b76

### CODE REVIEW

File: docs/pages/manage/actionsettings.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{% include navigation.html %}
+
+# Action Configuration
+
+Certain actions allow for a global configuration. These configurations are defined under the Action level.
+
```
Comment (NairiBonte) [inline docs/pages/manage/actionsettings.md:6]: This is quite high level -- Perhaps we could add more information as introduction? "Scripts can be easily build by defining and sequentially executing actions. These actions are defined through action types which allow for a global configuration. These configurations/types are defined...." -- just thinking a bit here

File: docs/pages/manage/generalsettings.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+The general IESI settings are configured in the `application.yml` file.
+
+## Home
+It is mandatory to define the root location of the IESI installation. This location is defined using the `home` key.
+
+```yaml
+iesi:
+  home: C:/Users/robbe.berrevoets/IESISandbox/0.4.0/b1
+  ...
```
Comment (NairiBonte) [inline docs/pages/manage/generalsettings.md:13]: Perhaps remove your name & replace through dummy?

File: docs/pages/manage/generalsettings.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  guard:
+    authenticate: N
+  ...
+```
+## Server
+The IESI installation can be run in server mode. If server mode is disabled, scripts are executed synchronously. If server mode is enabled, scripts are put in a queue and processed based on resource availibility. Server mode allows for concurrent execution of multiple scripts.
+
+Server mode is set by the `server.mode` key (`off` for disabled, `standalone` for enabled). If server mode is enabled, the following configuration has to be defined:
+* `server.threads.size`: the maximum number of concurrent executions of scripts
+* `server.threads.timeout`: the maximum duration (in minutes) a script can run. If the this time threshold is exceeded, the script execution is aborted.
```
Comment (NairiBonte) [inline docs/pages/manage/generalsettings.md:32]: *"if the this time..."

File: docs/pages/understand/understand.md [hunk 4]
```diff
@@ -26,7 +26,7 @@ The execution of an automation script can be associated with the following statu
 
 |Status|Description|
 |------|-----------|
-|ACTIVE|Script is running|
+|RUNNING|Script is running|
```
Comment (NairiBonte) [inline docs/pages/understand/understand.md:29]: killed?

Comment (BerrevoetsRobbe) [reply docs/pages/understand/understand.md:29]: KILLED is not a possible script run status. See core/java/iesi-core/src/main/java/io/metadew/iesi/common/configuration/ScriptRunStatus.java for all options. SKIPPED is omitted for now.

File: docs/pages/manage/actionsettings.md (hunk 1 above)
Comment (BerrevoetsRobbe) [reply docs/pages/manage/actionsettings.md:6]: Indeed, we can elaborate on this more in a next phase :)

**Pull request merged**